packages/manager/apps/dedicated/client/app/dedicated/server/dashboard/dashboard.routing.js

```diff
@@ -208,7 +208,7 @@ export default /* @ngInject */ ($stateProvider) => {
           .get(`/dedicated/technical-details/${serverName}`, {
             serviceType: 'aapi',
           })
-          .then(({ data }) => data?.baremetalServers)
+          .then(({ data }) => data?.baremetalServers?.storage)
```

Comment:
are you sure about that ?
having technicalDetails binded to data.baremetalServers.storage sounds wrong